tests/nnstreamer_filter_pytorch/runTest.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+##
+## @file runTest.sh
+## @author Parichay Kapoor <pk.kapoor@samsung.com>
+## @date May 7th 2019
+## @brief SSAT Test Cases for NNStreamer pytorch plugin
+##
+if [[ "$SSATAPILOADED" != "1" ]]
+then
+	SILENT=0
+	INDEPENDENT=1
+	search="ssat-api.sh"
+	source $search
+	printf "${Blue}Independent Mode${NC}
+"
+fi
+
+# This is compatible with SSAT (https://github.com/myungjoo/SSAT)
+testInit $1
+
+# Test constant passthrough custom filter (1, 2)
+PATH_TO_PLUGIN="../../build"
```

Comment:
Where is network model files of Squeezenet when the pytorch sub-plugin will be enabled?
Which package (e.g., RPM, DEB, etc) that put the model files for runTest.sh?

Reply:
Pytorch does not give model files to download (Squeezenet model in this case).
They are embedded in torchvision library (different from pytorch).
Currently, squeezenet model is added as part of this PR, but as myungjoo said, its too big (4.94MB). I have posted the possible solutions in comment below.

Reply:
Using MNIST model file (150kb), which are added as a PR.
No official place to download trained models for pytorch for MNIST.